packages/protocol/test/L1/SgxVerifier.t.sol

```diff
@@ -3,12 +3,68 @@ pragma solidity 0.8.24;
 
 import "./TaikoL1TestBase.sol";
 
-contract TestSgxVerifier is TaikoL1TestBase {
+// For SGX remote attestation
+import { AutomataDcapV3Attestation } from
+    "../../contracts/thirdparty/automata-attestation/AutomataDcapV3Attestation.sol";
+import { P256Verifier } from "../../lib/p256-verifier/src/P256Verifier.sol";
+import { SigVerifyLib } from
+    "../../contracts/thirdparty/automata-attestation/utils/SigVerifyLib.sol";
+import { PEMCertChainLib } from
+    "../../contracts/thirdparty/automata-attestation/lib/PEMCertChainLib.sol";
+
+import "../automata-attestation/utils/DcapTestUtils.t.sol";
+import "../automata-attestation/utils/V3JsonUtils.t.sol";
+
+contract TestSgxVerifier is TaikoL1TestBase, DcapTestUtils, V3JsonUtils {
+    address internal SGX_Y =
+        vm.addr(0x9b1bb8cb3bdb539d0d1f03951d27f167f2d5443e7ef0d7ce745cd4ec619d3dd7);
+    address internal SGX_Z = randAddress();
+    // For SGX remote attestation
+    AutomataDcapV3Attestation attestation;
+    SigVerifyLib sigVerifyLib;
+    P256Verifier p256Verifier;
+    PEMCertChainLib pemCertChainLib;
+    string internal constant tcbInfoPath = "/test/automata-attestation/assets/0923/tcbInfo.json";
+    string internal constant idPath = "/test/automata-attestation/assets/0923/identity.json";
+    string internal constant v3QuotePath = "/test/automata-attestation/assets/0923/v3quote.json";
+    bytes32 constant mrEnclave = 0x46049af725ec3986eeb788693df7bc5f14d3f2705106a19cd09b9d89237db1a0;
+    bytes32 constant mrSigner = 0xef69011f29043f084e99ce420bfebdfa410aee1e132014e7ceff29efa9659bd9;
+
     function deployTaikoL1() internal override returns (TaikoL1) {
         return
             TaikoL1(payable(deployProxy({ name: "taiko", impl: address(new TaikoL1()), data: "" })));
     }
 
+    function setUp() public override {
+        // Call the TaikoL1TestBase setUp()
+        super.setUp();
+
+        vm.warp(1_695_435_682);
```

Comment:
Maybe add a short comment of the reason why this is done, similar to what is done in `AutomataDcapV3AttestationTest`.